.contrib/docker/Dockerfile.web

```
@@ -18,4 +18,5 @@ RUN apt-get update \
 COPY requirements/*.txt ./requirements/
 ARG DJANGO_VERSION='==2.22.*'
 RUN pip install --no-cache-dir -r "requirements/dev.txt" "django${DJANGO_VERSION}"
+ENV PYTHONWARNINGS=error::RuntimeWarning,error::DeprecationWarning
```

```suggestion
ENV PYTHONWARNINGS=error
```

Let's try error always.